src/ray/core_worker/store_provider/memory_store/memory_store.cc

```diff
@@ -146,6 +150,7 @@ std::shared_ptr<RayObject> CoreWorkerMemoryStore::GetOrPromoteToPlasma(
   auto iter = objects_.find(object_id);
   if (iter != objects_.end()) {
     auto obj = iter->second;
+    obj->SetAccessed();
```

Comment:
Why do we need the accessed flag? I thought it would be enough to just throw the error below, if we Put the value but there's no ref to the object.

Reply:
Oh I see, for cases where the ref is still in scope but never used?

Reply:
Not sure I understand your comment, this is just recording the access.